packages/react/src/primitives/Menu/MenuItem.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { DropdownMenuItem } from '@radix-ui/react-dropdown-menu';
+
+import { ComponentClassNames } from '../shared/constants';
+import { MenuButton } from './MenuButton';
+import { MenuItemProps } from '../types';
+
+export const MENU_ITEM_TEST_ID = 'amplify-menu-item-test-id';
+
+export const MenuItem = React.forwardRef<
+  HTMLDivElement,
```

Comment:
It must be `HTMLDivElement` because the element where the ref is applied is an `HTMLDivElement`.

```
const DropdownMenuItem: React.ForwardRefExoticComponent<DropdownMenuItemProps & React.RefAttributes<HTMLDivElement>>
```

Reply:
Right, but there is no `div` get rendered in this use case. This will merge the original component props with the props of the supplied element/component and change the underlying DOM node. I think the ref will eventually go to the button, and pass in a `div` ref doesn't make sense I think